### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Charmis-Square**
added Script

### CODE REVIEW

File: react-native-in-app-payments-quickstart/README.md [hunk 1]
```diff
@@ -56,6 +56,17 @@ Change to the quick start sample folder and install the plugin using `yarn`:
 
 Run the React Native project from the `react-native-in-app-payments-quickstart` project folder:
 
+  1. First Add an script in build phase:
```
Comment (brandonjenniges) [inline react-native-in-app-payments-quickstart/README.md:59]: Change to "Add build phase to setup SquareInAppPaymentsSDK:"

File: react-native-in-app-payments-quickstart/README.md [hunk 2]
```diff
@@ -56,6 +56,17 @@ Change to the quick start sample folder and install the plugin using `yarn`:
...
+    1. Open the **Xcode** project for your application.
```
Comment (brandonjenniges) [inline react-native-in-app-payments-quickstart/README.md:60]: Change to "Open **RNSquareInAppPayments.xcworkspace**.

File: react-native-in-app-payments-quickstart/README.md [hunk 3]
```diff
@@ -56,6 +56,17 @@ Change to the quick start sample folder and install the plugin using `yarn`:
...
+    1. In the **Build Phases** tab for your application target, click the **+**
```
Comment (brandonjenniges) [inline react-native-in-app-payments-quickstart/README.md:61]: Change to "In the **Build Phases** tab for the **RNInAppPaymentsQuickstart** target click the **+**"

**Review by brandonjenniges — CHANGES_REQUESTED** [review]
Screenshot 2023-12-20 at 9 45 52 AM

The formatting looks off.

File: react-native-in-app-payments-quickstart/README.md (hunk 1 above)
Comment (Charmis-Square) [reply react-native-in-app-payments-quickstart/README.md:59]: Done !

**Review by brandonjenniges — APPROVED** [review]

**Pull request merged**